packages/nocodb/src/helpers/ncError.ts

```
@@ -0,0 +1,395 @@
+    super(param.message);
+    this.errors = param.errors;
+    this.humanReadableError = param.humanReadableError || false;
+  }
+
+  errors: ErrorObject[];
+}
+
+export class NcError extends NcErrorBase {
+  static _instance: NcError;
+  static get _() {
+    if (!this._instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)